rfc/website-revamp.md

```diff
@@ -0,0 +1,143 @@
+    * Contributor Experience Resource, such as code and documentation contribution guide and contribution best practices and tips
+    * Community Calendar
+    * Usage stats
+    * Governance & SIG info. This should be autogenerated from the Governance repo if possible. If not, we will continue to link to the Governance repo.
+* News
+    * Spinnaker newsletter
+    * Featured videos
+* Blog
+
+### Entrypoints & Tracks:
+Check out [the mock-up draft](https://docs.google.com/drawings/d/1FCzzvnK0a-_dQ5Nn1T4jz4DrXfr4tMzc9BrObMdEw3k/edit) to visualize the proposed content.
+
+*Note: tracks overlap just as personas overlap for real individuals, based on roles and experience*
+
+* Executive track: Should answer "What does my life look like if my team(s) use Spinnaker?" for Executives
+  * Card prompting clicks from those seeking to achieve business objectives, "Accelerate the velocity of your engineering teams"
+    * leads to a persona-tailored content link in Getting Started
+  * Case study content cards
+  * Who should use Spinnaker card
+
+* App Developer track: Should answer "What does my life look like if I use Spinnaker?" for AppDevs
+  * Card prompting appdevs to, "Take your code from commit to production in one pipeline"
+    * leads to a persona-tailored content link in Getting Started
+    * Video carousel
+    * Spinnaker Universe Calendar card
+    * Slack join card
+
+* Operator track: Should answer "What does my life look like if I use Spinnaker?" for Operators
+  * Card prompting operators to, "Create and operationalize a golden path to production for AppDevs"
+    * leads to a persona-tailored content link in Getting Started
+    * Video carousel
+    * Spinnaker Universe Calendar card
+    * Slack join card
+
+* Individual Contributor track: Should answer "What are my next steps?" for potential contributors
+  * Contributor card
+    * leads to Contributor section in the Community area
+  * Spinnaker Universe Calendar card
+
+### Documentation Features
+
+* Versioned documentation that corresponds to Spinnaker versions. This is a small widget in the top nav. See [K8s docs](kubernetes.io/docs/home) for an example.
+* Page-level table of contents that is always visible. This provides context and makes navigation within a page easier. See [Istio docs](https://istio.io/docs/concepts/what-is-istio/) for an example.
+
+## Drawbacks
+Will cost money, and cost needs to be negotiated amongst stakeholders.
+
+## Prior Art and Alternatives
+
+### Prior Art
+For design, Tensorflow does a very good job of creating clearly defined entryways for different roles. Additionally, other open source projects have implemented the documentation features we want.
+
+## Alternatives
+
+* Do nothing
+* Start completely over
+
+## Known Unknowns
+What do people outside of the Docs SIG want to see in a new Spinnaker.io?
+What is the impact of Spinnaker.io on the contributor experience and Spinnaker adoption?
+
+## Security, Privacy & Compliance
+
+n/a
```

Comment:
The above section notes that we'll start including usage stats on the site; might be worth commenting on that in this section (even if just to link our to the stats collection RFC).

Reply:
Thanks! We don't envision any new stats than what's already on spinnaker.io, so linking to the stats collection RFC should be good enough. Will add.